integration-tests/tests/step-functions-local.test.ts

```diff
@@ -0,0 +1,396 @@
+          })
+        );
+      });
+    });
+
+    describe("OAuth Url parameter not found", () => {
+      let stnContainerHelper: SfnContainerHelper;
+
+      beforeAll(async () => {
+        stnContainerHelper = new SfnContainerHelper();
+      });
+      afterAll(() => stnContainerHelper.shutDown());
+      it("should return OAuth Url parameter not Found", async () => {
+        // GIVEN
+        const input = JSON.stringify({ valid: "input" });
+        // WHEN
+        const responseStepFunction =
+          await stnContainerHelper.startStepFunctionExecution(
+            "OAuthURLParameterResourceNotFoundUnHappyTest",
+            input
+          );
+        const results = await stnContainerHelper.waitFor(
+          (event: HistoryEvent) =>
+            event?.type === "TaskStateExited" &&
+            event?.stateExitedEventDetails?.name === "Get OAuth URL",
+          responseStepFunction
+        );
+        // THEN
+        expect(results).toBeDefined();
+        expect(results?.length).toBe(1);
+        expect(results[0].stateExitedEventDetails?.output).toBe(
+          JSON.stringify({
+            Error: "Ssm.ParameterNotFoundException",
+            Cause:
+              "Service returned error code ParameterNotFound (Service: Ssm, Status Code: 400, Request ID: 4dc5822a-c05b-4f7d-a454-13b19c464876)",
+          })
+        );
+      });
+      it("should return OAuth Url parameter permission not configured", async () => {
+        // GIVEN
+        const input = JSON.stringify({ valid: "input" });
+        // WHEN
+        const responseStepFunction =
+          await stnContainerHelper.startStepFunctionExecution(
+            "OAuthURLParameterResourcePermissionUnHappyTest",
+            input
+          );
+        const results = await stnContainerHelper.waitFor(
+          (event: HistoryEvent) =>
+            event?.type === "TaskStateExited" &&
+            event?.stateExitedEventDetails?.name === "Get OAuth URL",
+          responseStepFunction
+        );
+        // THEN
+        expect(results).toBeDefined();
+        expect(results?.length).toBe(1);
+        expect(results[0].stateExitedEventDetails?.output).toBe(
+          JSON.stringify({
+            Error: "Ssm.SsmException",
+            Cause:
+              "User: arn:aws:sts::local:assumed-role/ola-otg-stack-OtgStateMachineRole-1SXH7VTMU9WPV/giJpYtNQeUFZRDAVegxozxZKXSxsRAoM is not authorized to perform: ssm:GetParameter on resource: arn:aws:ssm:eu-west-2:local:parameter/hmrc_oauth_url because no identity-based policy allows the ssm:GetParameter action (Service: Ssm, Status Code: 400, Request ID: 7ee4e39f-3fed-4b8f-afab-7b43c8c897bc)",
```

Comment:
What are we doing about the usage of the `ola-otg-stack`?

Reply:
This is the value defined currently in the Mocked file see https://github.com/alphagov/di-ipv-cri-otg-hmrc/blob/e59058bacf8595584acdc2f54e1b7e9b4715457b/integration-tests/tests/MockConfigFile.json#L204, it could have  been a more generic value though, I would suggest we merge this PR and clean up the Mock file to be more generic value i.e. remove the otg-stack references change it to something like otg-test-stack, run the test see it fail, update the tests make new PR merge it.